File: rename_to_auric.js
const fs = require('fs');
const path = require('path');

function walkDir(dir, callback) {
    fs.readdirSync(dir).forEach(f => {
        let dirPath = path.join(dir, f);
        let isDirectory = fs.statSync(dirPath).isDirectory();
        isDirectory ? walkDir(dirPath, callback) : callback(path.join(dir, f));
    });
}

let filesToUpdate = ['index.html'];
walkDir('src', function(filePath) {
    if (filePath.endsWith('.js') || filePath.endsWith('.jsx') || filePath.endsWith('.css')) {
        filesToUpdate.push(filePath);
    }
});

let changed = 0;
filesToUpdate.forEach(f => {
    if (fs.existsSync(f)) {
        let content = fs.readFileSync(f, 'utf8');
        let newContent = content.replace(/Aurem/g, 'Auric')
                                .replace(/aurem/g, 'auric')
                                .replace(/AUREM/g, 'AURIC');
        if (newContent !== content) {
            fs.writeFileSync(f, newContent, 'utf8');
            changed++;
        }
    }
});

console.log(`Done renaming! Modified ${changed} files.`);

File: rename_to_auric.py
import os

files_to_update = []
for root, _, files in os.walk('src'):
    for file in files:
        if file.endswith(('.js', '.jsx', '.css')):
            files_to_update.append(os.path.join(root, file))

files_to_update.append('index.html')

changed = 0
for f in files_to_update:
    if not os.path.exists(f): continue
    try:
        with open(f, 'r', encoding='utf-8') as file:
            content = file.read()
        if 'Aurem' in content or 'aurem' in content or 'AUREM' in content:
            new_content = content.replace('Aurem', 'Auric').replace('aurem', 'auric').replace('AUREM', 'AURIC')
            if new_content != content:
                with open(f, 'w', encoding='utf-8') as file:
                    file.write(new_content)
                changed += 1
    except Exception as e:
        print(f"Error processing {f}: {e}")

print(f"Done renaming! Modified {changed} files.")
